packages/babel-preset-react-app/create.js

```diff
@@ -46,6 +49,11 @@ module.exports = function(api, opts, env) {
     opts.absoluteRuntime,
     true
   );
+  // TODO: this needs to change.
+  // At this point we only create one babel config for all builds.
+  // We should be able to provide a babelconfig for every build.
+  // Does this mean we should move this to webpack or specify some BABEL_ENV?
```

Comment:
this has been done, the only thing that needs to change is us deciding if we want to opt-in or out by default